macos/generate_wazuh_packages.sh

```diff
@@ -185,6 +185,7 @@ function help() {
     echo "    -h, --help                    [  Util  ] Show this help."
     echo "    -i, --install-deps            [  Util  ] Install build dependencies (Packages)."
     echo "    -x, --install-xcode           [  Util  ] Install X-Code and brew. Can't be executed as root."
+    echo "    -v, --verbose                 [  Util  ] Show additional information during installation."
```

Comment:
This script does not install anything, it generates a package.

Reply:
Fixed here: ca4a97a3421ca8a4fde214fe596ac3af99de2806